Aptoide/appcoins-wallet-android
# [APPC-1094] Changed foreground release icon to fix zoomed icon

**What does this PR do?**

   Changes foreground of release adaptive icon

**Where should the reviewer start?**

- ic_launcher_foreground.xml (main)

**How should this be manually tested?**

  Use a device with an API 26 or above and check if the icon is similar to the one in dev

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/APPC-1094
